Sources/_StringProcessing/Regex/Core.swift

```diff
@@ -61,6 +61,13 @@ public struct Regex<Output>: RegexComponent {
   }
 }
 
+@available(SwiftStdlib 5.7, *)
+extension Regex {
+  public init(quoting string: String) {
+    self.init(node: .quotedLiteral(string))
+  }
+}
+
```

Comment:
I have a PR in flight soon that's adding doc comments, so we might be racing on this.

Reply:
sorry you are gonna have a conflict...